src/sim/sim_exit.hh

```diff
@@ -55,6 +55,21 @@ void exitSimLoop(const std::string &message, int exit_code = 0,
 void exitSimLoopNow(const std::string &message, int exit_code = 0,
                     Tick repeat = 0, bool serialize = false);
 
+void exitSimLoopWithHypercall(const std::string &message, int exit_code,
```

Comment:
As said above, I don't think we need these extra parameters here. But, let me know if I'm mistaken.